src/SA/get-netsession/Makefile

```diff
@@ -0,0 +1,15 @@
+BOFNAME := get-netsession
+COMINCLUDE := -I ../../common
+CC_x64 := x86_64-w64-mingw32-gcc
+CC_x86 := i686-w64-mingw32-gcc
+
+
+all:
+	$(CC_x64) -o $(BOFNAME).x64.o $(COMINCLUDE) -Os -c entry.c -DBOF 
+	$(CC_x86) -o $(BOFNAME).x86.o $(COMINCLUDE) -Os -c entry.c -DBOF
+	mkdir -p ../../../SA/$(BOFNAME) 
+	mv $(BOFNAME)*.o ../../../SA/$(BOFNAME)
+
```

Comment:
Noticed this one was built from the previous template, mind adding in the scanbuild, check, and clean targets from here: https://github.com/trustedsec/CS-Situational-Awareness-BOF/blob/master/src/base_template/Makefile

Reply:
Makefile should now match master